Source/FieldSolver/ImplicitSolvers/ImplicitSolver_fwd.H

```diff
@@ -0,0 +1,12 @@
+#ifndef _IMPLICIT_SOLVER_FWD_H_
+#define _IMPLICIT_SOLVER_FWD_H_
```

Comment:
This is not really a "forward declaration" file of C++ classes.

Instead, it is a file definining `ImplicitOptions` whole.

I would this call the file `ImplicitOptions.H`